docs/installation/integrations/drupal.md

```diff
@@ -0,0 +1,74 @@
+---
+menu-title: Drupal
+meta-title: Real-time editing in Drupal | CKEditor 5 documentation
+category: integrations
+order: 90
+modified_at: 2023-10-06
+---
+
+{@snippet installation/integrations/framework-integration}
+
+# Real-time editing in Drupal
+
+Drupal is an open-source editing platform. CKEditor 5 is the default editor module for Drupal. The [CKEditor 5 Premium Features module](https://www.drupal.org/project/ckeditor5_premium_features) provides instant integration of the real-time collaboration features. Currently, the module includes a set of GDocs-style content collaboration tools, the {@link features/productivity-pack Productivity Pack} of essential formatting and navigation features, plus document import and export plugins.
+
+The module also includes the free Full Screen Mode, a free-to-use plugin that maximizes the editing area. It is especially useful when using features like Document outline or Comments which take up extra space around the editor.
+
+## Real-time collaboration in Drupal with CKEditor 5
+
+With collaboration features, users can work together to write, review, and discuss their content right within Drupal. No need to use different apps for drafting and commenting &ndash; you can do it all in one place.
+
+{@img assets/img/drupal.png 1876drupal Drupal with CKEditor&nbsp;5 real-time editing.}
```

Comment:
Can we make/get a gif (or a vid)? I usually against them, but they can easily explain real-time visually.

Reply:
I reused the one from Premium module guide.